### PR TITLE
Mention RP2040 on drag-and-drop installation instructions

### PR DESCRIPTION
**Pull request by pdxlocations**
This PR adds RP2040/RAK11300 to the drag and drop instruction pages.
[download page](https://sandbox-git-rp2040-drag-and-drop-pdxlocations.vercel.app/downloads)
[flashing options](https://sandbox-git-rp2040-drag-and-drop-pdxlocations.vercel.app/docs/getting-started/flashing-firmware/nrf52/)
[flashing instructions](https://sandbox-git-rp2040-drag-and-drop-pdxlocations.vercel.app/docs/getting-started/flashing-firmware/nrf52/drag-n-drop)
[nRF/RP2040 Factory Erase](https://sandbox-git-rp2040-drag-and-drop-pdxlocations.vercel.app/docs/getting-started/flashing-firmware/nrf52/nrf52-erase)

### CODE REVIEW

File: src/pages/downloads/index.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ const Firmware = (): JSX.Element => {
               </div>
```
Comment (GUVWAF) [inline src/pages/downloads/index.tsx:64]: Maybe here one line 63 as well? "nRF52/RP2040".

Comment (pdxlocations) [reply src/pages/downloads/index.tsx:64]: good catch

**Comment by pdxlocations** [comment]
added [RP2040 Factory Erase](https://sandbox-git-rp2040-drag-and-drop-pdxlocations.vercel.app/docs/getting-started/flashing-firmware/nrf52/rp2040-erase)

File: docs/getting-started/flashing-firmware/nrf52/drag-n-drop.mdx [hunk 2]
```diff
@@ -16,7 +16,7 @@ Download and unzip the latest firmware from [Meshtastic Downloads](https://mesht
 1. Connect your device to your computer with a USB data cable.
 2. Double click the reset button on your device (this will put it into bootloader mode)
```
Comment (GUVWAF) [inline docs/getting-started/flashing-firmware/nrf52/drag-n-drop.mdx:17]: For RP2040 you should actually hold the BOOTSEL button while plugging in the cable. Double click doesn't seem to work.

**Review by GUVWAF — APPROVED** [review]
Great, thanks!

**Review by caveman99 — APPROVED** [review]

**Pull request merged**